### ISSUE TITLE
Fatal error: Uncaught Error: Call to undefined function pam_auth()

### ISSUE THREAD

**Issue by aliorhun** [the-kbA-team/php-pam#4]
When i was use this modul, I have an error: "Fatal error: Uncaught Error: Call to undefined function pam_auth()". Thank you.

/var/www/html/a.php:
```
<?php
pam_auth("deneme","1")
?>
```
/etc/php/7.0/apache2/php.ini: [ADDED]
> pam.servicename = "php";

/etc/pam.d/php: [CREATED]
> auth    sufficient  /lib/security/pam_pwdb.so shadow nodelay
> account sufficient  /lib/security/pam_pwdb.so

Apache Log:
> [Mon Nov 05 10:34:29.597243 2018] [:error] [pid 1392] [client 10.67.100.1:48860] PHP Fatal error:  Uncaught Error: Call to undefined function pam_auth() in /var/www/html/a.php:5\nStack trace:\n#0 {main}\n  thrown in /var/www/html/a.php on line 5

**Comment by gregor-j** [comment]
Is the compiled PHP module enabled?
```
php -m  | grep pam
```
If not:
```
phpenmod pam
```

**Comment by aliorhun** [comment]
When i run "**phpenmod pam**", it takes the warning messages. 
> WARNING: Module pam ini file doesn't exist under /etc/php/7.0/mods-available
> WARNING: Module pam ini file doesn't exist under /etc/php/7.0/mods-available
> WARNING: Module pam ini file doesn't exist under /etc/php/7.0/mods-available

System doesn't include "**pam.ini**" in everywhere. (updatedb & locate pam.ini) Pam extension(pam.so) installed on "/usr/lib/php/20151012". Make building was succesfull.

**Comment by gregor-j** [comment]
```
echo "extension=pam.so" > /etc/php/7.0/mods-available/pam.ini
```

This is a **very** basic question regarding the general functionality of PHP modules and definitly not regarding this module specifically. Furthermore, the information I gave you is easily available using a searchengine of your choice. This leads me to the conclusion, that even after I gave you the hint, that your module is not enabled, you were too lazy to do a basic web search, but let me do the work instead. 

I'm closing this issue.
